roboteam_ai/src/skills/Skill.cpp

```diff
@@ -20,7 +21,13 @@ Skill::Skill(std::string name, bt::Blackboard::Ptr blackboard) : bt::Leaf(std::m
 
 }
 void Skill::publishRobotCommand(roboteam_msgs::RobotCommand cmd) {
-    ioManager.publishRobotCommand(cmd);
+    ros::NodeHandle nh;
+    std::string ourSideParam;
+    nh.getParam("our_side",ourSideParam);
+    if(ourSideParam=="right"){
+        rotateRobotCommand(cmd);
```

Comment:
This is a c-style solution which is fast but not very intuitive in readability.

Reply:
I fixed this in new commits